src/views/ScopesetExpander/index.jsx

```diff
@@ -10,15 +10,29 @@ import Tooltip from '@material-ui/core/Tooltip';
 import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
 import ListItemText from '@material-ui/core/ListItemText';
+import IconButton from '@material-ui/core/IconButton';
 import ArrowExpandVerticalIcon from 'mdi-react/ArrowExpandVerticalIcon';
+import CompareIcon from 'mdi-react/CompareIcon';
+import PlusIcon from 'mdi-react/PlusIcon';
+import DeleteIcon from 'mdi-react/DeleteIcon';
 import LinkIcon from 'mdi-react/LinkIcon';
 import Dashboard from '../../components/Dashboard/index';
 import Button from '../../components/Button';
 import splitLines from '../../utils/splitLines';
 import scopesetQuery from './scopeset.graphql';
+import ScopesetDiff from '../../components/ScopesetDiff';
 
 @hot(module)
 @withStyles(theme => ({
+  iconButton: {
+    '& svg': {
+      fill: theme.palette.text.primary,
+    },
+    marginRight: -theme.spacing.triple,
+    [theme.breakpoints.down('sm')]: {
+      marginRight: -14,
```

Comment:
`marginRight: -theme.spacing.double,`

Reply:
Sorry if I wasn't clear. This was to replace `marginRight: -14,`, not `marginRight: -theme.spacing.double,`. The gaol is to minimize hardcoded values.

Reply:
Oh, sorry :grimacing: .
Just to get it right this time,
we just want `marginRight: -theme.spacing.double,`
and no `[theme.breakpoints.down('sm')]`
right?

Reply:
```jsx
[theme.breakpoints.down('sm')]: {
  marginRight: -theme.spacing.double,
},
```